pkgs/development/tools/misc/igprof/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchFromGitHub, cmake}:
+
+stdenv.mkDerivation rec {
+  pname = "igprof";
+  version = "5.9.16";
+
+  src = fetchFromGitHub {
+    owner  = pname;
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0glpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = {
+    description = "The Ignominous Profiler";
+
+    longDescription = ''
+      IgProf is a simple nice tool for measuring and analysing application
+      memory and performance characteristics. IgProf requires no changes to the
+      application or the build process.
+    '';
+
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = with maintainers; [ ktf ];
```

Comment:
This doesn't evaluate since `maintainers` is an attribute set in `stdenv.lib`. A typical pattern to use `with maintainers` is to use `meta = with stdenv.lib` which makes all attributes from `stdenv.lib` available in the `meta` block.